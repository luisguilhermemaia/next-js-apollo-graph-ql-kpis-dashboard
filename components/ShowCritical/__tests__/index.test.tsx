import '@testing-library/jest-dom/extend-expect';
import { RenderResult, render } from '@testing-library/react';
import React from 'react';
import { ShowCritical } from '../index';

describe('ShowCritical', () => {
  let renderResult: RenderResult;

  const doRender = () => {
    renderResult = render(<ShowCritical setState={jest.fn()} />);
  };

  it('renders correctly', () => {
    doRender();

    const { container } = renderResult;

    expect(container).toMatchSnapshot();
  });
});
