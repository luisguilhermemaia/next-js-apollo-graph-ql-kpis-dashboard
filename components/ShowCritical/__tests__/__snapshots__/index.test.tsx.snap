// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowCritical renders correctly 1`] = `
<div>
  <div
    class="ShowCritical__StyledContainer-sc-1m229mh-0 dSBfNi"
  >
    <input
      type="checkbox"
    />
    <label
      for="critical-checkbox"
    >
      Show critical
    </label>
  </div>
</div>
`;
